packages/babel-traverse/src/scope/lib/renamer.js

```diff
@@ -2,9 +2,33 @@ import Binding from "../binding";
 import splitExportDeclaration from "@babel/helper-split-export-declaration";
 import * as t from "@babel/types";
 
+function isInFunctionParameterScope(path) {
+  const computableNode = path.findParent(path => {
+    return (
+      path.type === "AssignmentPattern" ||
```

Comment:
WDYT about using `path.parentPath.isAssignmentPattern({ right: path.node })` instead?
I don't think that it would change anything, but the `left` of an assignment isn't computable anyway.

Reply:
> WDYT about using path.parentPath.isAssignmentPattern({ right: path.node }) instead?

Good point. I have refactored this part and feel horrible on what I was writing a month ago.